cloudbuild-stackdriver.yaml

```diff
@@ -0,0 +1,11 @@
+steps:
+- name: 'debian:jessie'
+  dir: spinnaker-monitoring-daemon
+  entrypoint: "bash"
+  args: ["-c", "echo 'google-api-python-client' >> requirements.txt; echo 'oauth2client' >> requirements.txt"]
+- name: 'gcr.io/cloud-builders/docker'
+  dir: spinnaker-monitoring-daemon
+  args: ["build", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME:latest", "-f", "Dockerfile", "."]
```

Comment:
These should definitely have the solution type in the tag, otherwise it'll be a last-write wins scenario. I suggest
```
gcr.io/$PROJECT_ID/$REPO_NAME:stackdriver-$COMMIT_SHA
gcr.io/$PROJECT_ID/$REPO_NAME:stackdriver-latest
```